.github/workflows/people.yml

```diff
@@ -53,6 +53,35 @@ jobs:
             exit 1
           fi
 
+  build-contact:
```

Comment:
should we call the jobs `build-back-{app}` and `build-front-contact`?

Reply:
https://github.com/numerique-gouv/people/blob/c0969b410baa97ae31975501b28198613206977a/.github/workflows/people.yml#L61